src/commonmark/en/content/developer/web-api.md

```diff
@@ -3601,6 +3601,11 @@ The import process can be customized using a set of import parameters:
 <td>false | true</td>
 <td>Indicates whether the import should be done asynchronous or synchronous. The former is suitable for very large imports as it ensures that the request does not time out, although it has a significant performance overhead. The latter is faster but requires the connection to persist until the process is finished.</td>
 </tr>
+<tr class="even">
+<td>force</td>
+<td>false | true</td>
+<td>Indicates whether the import should be forced. Data import could be rejected for various reasons of data set locking for example due to approval, data input period, expiry days and etc. In order to override such locks and force data input one can use data import with force=true. However, one needs to be a **superuser** for this parameter to work.</td>
```

Comment:
typo: change "expiry days and etc." to "expiry days, etc."

Reply:
Fixed.